src/main/java/DbConnection/DataSourceInit.java

```diff
@@ -0,0 +1,40 @@
+package DbConnection;
+
+import com.mchange.v2.c3p0.ComboPooledDataSource;
+import temp.Market;
+
+import java.beans.PropertyVetoException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+
+public final class DataSourceInit {
+    private static ComboPooledDataSource instance;
+
+    public static ComboPooledDataSource getDataSource() {
```

Comment:
Use interfaces here. You don't need to return concrete class. Other code shouldn't know about your implementation.